.github/ISSUE_TEMPLATE/where-to-report-issues.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "⚠️ Where to report issues ?"
+about: File issues on the main Eclipse Che repository at https://github.com/eclipse/che/issues
+title: Issues need to be filed on the main Eclipse Che repository
+labels: ''
+assignees: ''
+
+---
+
+## Where to report issues?
+
+This repository is not the primary repository of Eclipse Che.
+
+🚨 Please don't submit new issues here. 🚨
```

Comment:
```suggestion
🚨 Please don't submit new issues that are not related to documentation in this repository. 🚨
```

Reply:
@rkratky : getting consistent with the others Che repositories, we want to have a single place to consolidate the backlog and make it easier to the community to look for issues/feedbacks/enhancements. There is a kind/docs label on eclipse/che repo which should be used appropriately.

Reply:
@slemeur I find it highly unintuitive to file issues about the content of a repo in a different repo. It also prevents us from fully using GH features that simplify work on PRs and issues when handled 'normally', i.e. through the issue queue of respective repos.

Reply:
here I would still say that it's "no issues at all" as it was decided that all issues should be in `eclipse/che` repository

Issues should have `area/doc` label in `eclipse/che`

Reply:
> it was decided

Where?

Reply:
@rkratky at the F2F during End Game discussion
All issues have to be handled in only one repository: Eclipse Che (https://github.com/eclipse/che)

Reply:
> at the F2F during End Game discussion

Heh, I must've dozed off... :(

>  All issues have to be handled in only one repository: Eclipse Che (https://github.com/eclipse/che)

OK, I get it, thanks (I find that to be an unfortunate decision, but too late now).

Will somebody be send an e-mail to che-dev about this?

Reply:
I'm not sure it was acted to send a separate mail but yes I can send it

do you think it's better after merging all these `github redirect` PR (and on other repositories) or I should send it as soon as possible ?

Reply:
I'd send it as soon as possible. It's a significant process change.

Reply:
mail sent

Reply:
The whole point is simplicity and consistency for the community and users. 
If someone wants to create an issue, there will be a single backlog which give an easier way to get visibility on what's going on. The use of labels and milestone will be easier to manage too. 
If someone is looking at all the issues resolved for a given milestone, until having a single backlog, he/she would have to do a request across dozen of repositories. 
It is also very inconvenient when someone is looking at if an issue already exist.

One last point, we currently have issues for docs on github.com/eclipse/che repo, I'm doubtful we've been able to manage these two spread backlogs in sync.

Reply:
I understand the reasoning (and I agree with it). But I believe the wrong tool was chosen. GH issues are specifically tailored for working with respective repositories. Tracking issues for multiple repos in one issue queue doesn't scale. As a general-purpose issue tracker, GH issues sucks.

For such purpose, an external issue tracker should be used. GH issues is severely limited in functionality -- for example, the inability to filter issue notification mails in any way makes it impossible track project areas efficiently. One always needs to open the list of issues on the web just to check if new issues were created for a particular label.

Reply:
That's a fair point @rkratky. 
For the committers it will require a bit more effort, but for the community and users, it will become cleaner and simpler.